eox_core/cms/views.py

```diff
@@ -31,30 +35,36 @@ def management_view(request):
     """
     if 'text/html' in request.META.get('HTTP_ACCEPT', '') and request.method == 'GET':
```

Comment:
since we already have a `@require_http_methods(["GET"])`, we could remove `request.method == 'GET'`